src/Credentials.js

```diff
@@ -145,21 +147,31 @@ class Credentials {
   *  Send a push notification to a user, consumes a token which allows you to send push notifications
   *  and a url/uri request you want to send to the user.
   *
-  *  @param    {String}                  token       a push notification token (get a pn token by requesting push permissions in a request)
+  *  @param    {String}                  token              a push notification token (get a pn token by requesting push permissions in a request)
+  *  @param    {String}                  pubEncKey          the public encryption key of the receiver, encoded as a base64 string
+  *  @param    {String}                  payload.url        a uport request url
```

Comment:
add doc line above payload properties/keys (just formats it a bit better) - @param {Object}     payload      push notification payload